syntax ="proto3";
//创建rpc命名空间
package RPC;
//生成service类必须的
option cc_generic_services=true;
//错误码类
message FriendResultCode
{
    int32 errcode=1;
    bytes errmsg=2;
}

//注册请求
message GetFriendListRequest
{
    uint32 userid=1;
    
}
//注册响应
message GetFriendListResponse
{
    FriendResultCode result=1;
    repeated bytes friends=2;
}
//RPC函数
service FriendServiceRpc
{
    rpc GetFriendList(GetFriendListRequest)returns(GetFriendListResponse);
}